waspc/src/Lexer.hs

```diff
@@ -64,7 +61,7 @@ reservedNames =
     , reservedNameRoute
     , reservedNameEntity
     , reservedNameEntityForm
-    , reservedNameButton
+    , reservedNameQuery
```

Comment:
I forgot to add this in previous PRs.

Reply:
ah yes, I think I also forgot to add something earlier, now you reminded me :D